app/src/main/java/org/mozilla/focus/fragment/BrowserFragment.kt

```diff
@@ -240,17 +244,23 @@ class BrowserFragment :
         }
     }
 
+    private fun customizeFindInPage(view: View) {
+        val findInPageBar = view.findViewById<FindInPageBar>(R.id.find_in_page)
+        val newParams = findInPageBar.layoutParams as CoordinatorLayout.LayoutParams
+        newParams.gravity = Gravity.BOTTOM
```

Comment:
Seems that some updates were made. @brampitoyo At this [Figma link](https://www.figma.com/file/2V6wgf9fRFdYfLaf5c3oPj/Product-UI?node-id=2133%3A282824) we can find Find in page bar at two differerent positions. Can you tell us which one is the correct one?

![Screenshot 2021-08-12 at 10 08 01](https://user-images.githubusercontent.com/11731652/129153700-cfca57c2-6d61-427a-a86f-003a796e6687.png)

![Screenshot 2021-08-12 at 10 09 29](https://user-images.githubusercontent.com/11731652/129153707-6c5a754c-4c42-4a12-a5c4-f15693a6d6a8.png)

Reply:
In the interest of avoiding this PR from going stale, let's land this while we wait for final UX review. Updating this change shouldn't be difficult after it lands.

Reply:
@ionutbedregeanu @jonalmeida We should use the top-aligned “Find in page” UI.

When the toolbar is moved to the bottom of the screen (maybe we will plan for this in the future), then the “Find in page” UI will also move to the bottom, so it will be located above the keyboard.